src/templates/ja/2019/index.html

```diff
@@ -1,9 +1,9 @@
 {% extends "base/2019/index.html" %}
 
-{% block title %}The {{ year }} Web Almanac{% endblock %}
+{% block title %} {{ year }} Web Almanac{% endblock %}
 {% block description %}Web Almanacは、Webコミュニティの専門知識とHTTP Archiveのデータやトレンドを組み合わせた、毎年恒例のWeb状態のレポートです。{% endblock %}
 
-{% block twitter_image_alt %}The {{ year }} Web Almanac{% endblock %}
+{% block twitter_image_alt %} {{ year }} Web Almanac{% endblock %}
```

Comment:
Nit, but ca we remove the unnecessary space?
```suggestion
{% block twitter_image_alt %}{{ year }} Web Almanac{% endblock %}
```